index.html

```diff
@@ -2999,32 +2999,56 @@ <h3>Trust Model</h3>
         <ul>
           <li>
 The [=verifier=] trusts the [=issuer=] to issue the [=credential=] that
-it received. To establish this trust, a [=credential=] is expected to either:
+it received after verification has been performed. 
+To establish this trust, a [=credential=] is expected to either:
             <ul>
               <li>
-Secure the [=credential=] with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
+Be secured with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
 [=issuer=] generated the [=credential=] (that is, it is a
 [=verifiable credential=]), or
               </li>
               <li>
 Have been transmitted in a way clearly establishing that the [=issuer=]
-generated the [=verifiable credential=] and that the
-[=verifiable credential=] was not tampered with in transit or storage. This
+generated the [=credential=] and that the
+[=credential=] was not tampered with in transit or storage. This
 trust could be weakened depending on the risk assessment of the [=verifier=].
               </li>
             </ul>
           </li>
           <li>
 All [=entities=] trust the [=verifiable data registry=] to be
 tamper-evident and to be a correct record of which data is controlled by which
-[=entities=].
+[=entities=]. This is typically achieved by the method of its publication. 
+This could be by a peer to peer protocol from a trusted publisher, or 
+on a publicly accessible and well known web site (with a content hash), 
+or on a blockchain etc. When entities publish meta data about themselves
+they can be integrity protected by signing with the entity's private key.
           </li>
           <li>
 The [=holder=] and [=verifier=] trust the [=issuer=] to issue
 true (that is, not false) [=credentials=] about the [=subject=], and to
 revoke them quickly when appropriate.
           </li>
           <li>
+The [=holder=] may trust the [=issuer=] because the [=holder=] already
+has a pre-existing trust relationship with the [=issuer], for example, 
+an employee is provided with an employment [=verifiable credential=] by 
+an employer, or a citizen is issued with an electronic passport by 
+their government. 
```

Comment:
```suggestion
The [=holder=] might trust the [=issuer=]'s [=claims=] because the [=holder=]
has a pre-existing trust relationship with the [=issuer=]. For example, an
employer might provide an employee with an employment [=verifiable credential=],
or a government might issue an electronic passport to a citizen. 
```